SOURCES/xapi-1.249.32-allow-a-user-to-select-on-which-SR-to-run-quicktest.backport.patch

```diff
@@ -0,0 +1,95 @@
+From 5a07265ef43a3d50b586ca029759ca0638705bcb Mon Sep 17 00:00:00 2001
+From: BenjiReis <benjamin.reis@vates.fr>
+Date: Tue, 18 Jul 2023 13:24:18 +0200
+Subject: [PATCH] Allow a user to select on which SR to run quicktest
```

Comment:
If you had to adapt the patch for it to apply, you should add a comment on top of this, explaining your changes.